src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -48,6 +48,7 @@ export const validateEncryptSubmissionParams = celebrate({
         }),
       )
       .optional(),
+    paymentProducts: Joi.array().items(Joi.object()),
```

Comment:
should we list out the object fields?
```suggestion
    paymentProducts: Joi.array().items(
      Joi.object().keys({
        data: (...same as those in `admin-form.payments.controller.ts`), 
        selected: Joi.boolean()
        quantity: JoiInt.positive().required()
      }),
    ),
```

Reply:
Great suggestion, yes we should. 89571be7 also synchronized over to `admin-form.payments.controller.ts`'s `updatePaymentSettings`